.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,7 +11,8 @@
 <!-- Explain, without going into the details, what this PR does, and what problem it solves. -->
 
 **Which issue(s) this PR fixes**:
-<!-- Add a link to the JIRA issue. Otherwise, put "no issue."
+<!-- Add a link to the JIRA issue. Otherwise, put "no issue." 
+If this solves a COPS issue, you must include release notes below.
```

Comment:
Release notes are required, regardless. If there are no customer-facing changes, it's still required even if the content is "none"

Reply:
What I'm suggesting here is if the fix is for cops then release notes cannot be "none"